egs2/vctk/asr1/conf/decode_asr.yaml

```diff
@@ -0,0 +1,3 @@
+lm_weight: 0.6
+ctc_weight: 0.4
+beam_size: 60
```

Comment:
I found the decoding beam size is larger than our usual recipe. Is this intentional?

Reply:
done